font_scale_plugin/src/main/java/pl/droidsonroids/foqa/deviceinfo/FontScaleInteractor.kt

```diff
@@ -30,12 +31,17 @@ internal class FontScaleInteractor(private val context: Context) {
     @RequiresApi(Build.VERSION_CODES.M)
     private fun startSystemSettingsActivity() {
         val intent = Intent(Settings.ACTION_MANAGE_WRITE_SETTINGS)
-                .addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
-                .setData(Uri.parse("package:" + context.packageName))
+            .addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+            .setData(Uri.parse("package:" + context.packageName))
         try {
             context.startActivity(intent)
         } catch (e: ActivityNotFoundException) {
-            Toast.makeText(context, R.string.foqa_font_scale_system_settings_disabled, Toast.LENGTH_SHORT).show()
+            Log.w("FoQA", e)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/DroidsOnRoids/FoQA/blob/33947cbd43582f80a9c240ce3d0ec15a695e8916/font_scale_plugin/src/main/java/pl/droidsonroids/foqa/deviceinfo/FontScaleInteractor.kt#L39"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>